lib/ProductOpener/Users.pm

```diff
@@ -949,6 +969,27 @@ sub remove_user_by_org_admin ($orgid, $user_id) {
 	return;
 }
 
+sub add_users_to_org_by_admin ($org_id, $email_list) {
+
+	# Convert the email_list into an array of email addresses
+	my @emails = split(/,\s*/, $email_list);
+
+	foreach my $email (@emails) {
+		# Check if the email is associated with an OpenFoodFacts account
+		my $user_id = is_email_has_off_account($email);
+		if (defined $user_id) {
+			# Add the user to the organization
+			add_user_to_org($org_id, $user_id, ["members"]);
+		}
+		else {
+			# we can send an invitation email to the user
+			return 0;
```

Comment:
If you use "return", the function will exit immediately, and the other e-mail addresses after that one won't be processed.

Reply:
Yes, right as of now if one of the mails in the list is not having off acc, then other mails after that won't be processed.